Zend/zend_operators.h

```diff
@@ -414,6 +414,17 @@ ZEND_API int ZEND_FASTCALL string_compare_function(zval *op1, zval *op2);
 ZEND_API int ZEND_FASTCALL string_case_compare_function(zval *op1, zval *op2);
 ZEND_API int ZEND_FASTCALL string_locale_compare_function(zval *op1, zval *op2);
 
+/* NOTE: The locale-independent alternatives to ctype(isalpha/isalnum) were added to fix bugs in php 8.0 patch releases, and should not be used externally until php 8.2 */
+ZEND_API extern const bool zend_isalnum_map[256];
+
+static zend_always_inline bool zend_isalpha_ascii(unsigned char c) {
+	/* Returns true for a-z and A-Z in a locale-independent way.
+	 * This is implemented in a way that can avoid branching. Note that ASCII 'a' == 'A' | 0x20. */
+	c = (c | 0x20) - 'a';
+	return c <= ('z' - 'a' + 1);
```

Comment:
Shouldn't this be 
```suggestion
	return c < 'z' - 'a';
```
?

Reply:
Fixed